lib/rucio/client/baseclient.py

```diff
@@ -167,7 +167,7 @@ def __init__(self,
                 self.logger.debug('HTTPS is required, but no ca_cert was passed and X509_CERT_DIR is not defined. Trying to get it from the config file.')
                 try:
                     self.ca_cert = path.expandvars(config_get('client', 'ca_cert'))
-                except (NoOptionError, NoSectionError):
+                except (NoOptionError, NoSectionError, ConfigNotFound):
                     self.logger.debug('No ca_cert found in configuration. Falling back to Mozilla default CA bundle (certifi).')
```

Comment:
I think the logger message should be different in the event of a `ConfigNotFound`. Right now it'd log:
> 'No ca_cert found in configuration. ...'

But in the case of `ConfigNotFound`, it'd be more accurate to say "No configuration found."